src/application.ts

```diff
@@ -28,9 +36,11 @@ export default class Application {
   async executeOperations(ops: Operation[]): Promise<OperationResponse[]> {
     return await this.createTransaction(
       ops.map(async op => {
-        const processor = this.processorFor(op);
-        const result = await processor.execute(op);
-        return this.buildOperationResponse(result);
+        const processor = this.processorFor(op.ref.type);
+        const data = await processor.execute(op);
+        const included = await this.resolveIncludes(op, data);
+
```

Comment:
here is probably where you want to now mutate the resource objects in `data` to "inject" the `relationships` object that provides the links to the resources now in `included`